cli/types/cypress.d.ts

```diff
@@ -780,15 +780,15 @@ declare namespace Cypress {
      *
      * @see https://on.cypress.io/clearcookie
      */
-    clearCookie(name: string, options?: Partial<Loggable & Timeoutable>): Chainable<null>
+    clearCookie(name: string, options?: Partial<CookieOptions>): Chainable<null>
 
     /**
      * Clear all browser cookies.
      * Cypress automatically clears all cookies before each test to prevent state from being shared across tests. You shouldn't need to use this command unless you're using it to clear a specific cookie inside a single test.
      *
      * @see https://on.cypress.io/clearcookies
      */
-    clearCookies(options?: Partial<Loggable & Timeoutable>): Chainable<null>
+    clearCookies(options?: Partial<CookieOptions>): Chainable<null>
```

Comment:
When domain is not specified, does this clear cookies in all domains?

Reply:
No, current superdomain. Updated the comment in https://github.com/cypress-io/cypress/pull/24471/commits/648449c6c507392bcee2a43f5cfba3d3225307c0.